lib/modules/manager/flux/extract.ts

```diff
@@ -51,7 +58,7 @@ function readManifest(content: string, file: string): FluxManifest | null {
           resource.apiVersion?.startsWith('helm.toolkit.fluxcd.io/') &&
           resource.spec?.chart?.spec?.chart
         ) {
-          manifest.releases.push(resource);
+          manifest.helmReleases.push(resource);
```

Comment:
we need to use a single releases variable, otherwise we get the deps in wrong order and the auto replace will fail.

so all deps need to stay in same order as they occured inside the file.

Reply:
yes, sorry. I mean updatable dependencies like helm release or git repo.